alphastats/dataset/dataset.py

```diff
@@ -449,6 +449,7 @@ def plot_volcano(
             rawinput=self.rawinput,
             metadata=self.metadata,
             preprocessing_info=self.preprocessing_info,
+            feature_repr=self._feature_to_repr_map,
```

Comment:
(nit) it helps keeping tracking of things by sticking to a name :-) `feature_to_repr_map=self._feature_to_repr_map`